mapping/mapping_vectors.go

```diff
@@ -121,6 +134,27 @@ func processVector(vecI interface{}, dims int) ([]float32, bool) {
 	return rv, true
 }
 
+func processVectorBase64(vecBase64 interface{}) (interface{}, bool) {
```

Comment:
The code within here looks exactly as in `document.DecodeVector(..)`. Let's just reuse that here after you do the type assertion below - where you can interpret an `err != nil` as false.